docs/assets/commonMarkdown/sharedDataCNconf.md

```diff
@@ -0,0 +1,20 @@
+
+**Before starting CNs**, add the following configuration items in the CN configuration file **cn.conf**:
+
+```Properties
+starlet_port = <starlet_port>
+storage_root_path = <storage_root_path>
+```
+
+#### starlet_port
+
+The CN heartbeat service port for the StarRocks shared-data cluster. Default value: `9070`.
+
+#### storage_root_path
```

Comment:
This is a must have configuration in v3.0 but optional for `CN` since v3.1. If the end user want a completely stateless CN deployment, `storage_root_path` must be explicitly set to empty in configuration.
`storage_root_path = `
because the default value for it is `$STARROCKS_HOME/storage`.

and in such case, all the cache ability of tablets assigned to this CN node will be automatically disabled no matter what the table/partition property settings.

Reply:
Thanks @kevincai 
When would someone want a completely stateless CN deployment? (When is a cache not desired?)

Reply:
There are cases that the user may want to  use a diskless CN.
1) in kubernetes env, a container without storage volume attached, usually is lightweight than the one with storage volume, and thus easily to acquire and release on demand.
2) under certain scenario, shared-data mode can be used as data lake with all cold data stored in remote storage, mostly can think it as archiving, more and more data are ingested, but the query is rarely and when there is one, often, the query may be a one-time shot.
3) in our v3.2 multi-warehouse mode, user can group their activities into different warehouses (a warehouse is a group of cn nodes), activities such as data ingestion and compaction, can be done directly against the remote storage, as the input will be expired anyway (replaced by later compaction, or the query is not among the same cn groups). In such case, the disk cache is optional and opt to absent.